### ISSUE TITLE
Conflict with More Milestone +100

### ISSUE THREAD

**Issue by Tropelage** [cepheidqq/FactorySkyline#30]
There seems to be an issue going on with this other mod. Every time one tries to open a blueprint from the menu "R" the program crashes. I tried an earlier version of the mod causing the problem and it works, but not with the latest More Milestones + 100

**Comment by cepheidqq** [comment]
Which version will cause the crash?

**Comment by StretchyAlien** [comment]
I'm also having this crash.  Relevant details:

Build: 138299 (EA)
MoreMilestones: 0.7.11
FactorySkyline: 0.1.2

Still crashes when attempting to use a blueprint.

**Comment by cepheidqq** [comment]
I‘ve got the case which causes a crash. I will fix it...

**Comment by Tropelage** [comment]
Yeah this is why I lowered the version of MoreMilestone +100 to 0.7.10 and it does not crash.

**Comment by cepheidqq** [comment]
The 0.1.3 has been uploaded. It's an interesting mod, I've played to war 10...

**Comment by Tropelage** [comment]
The More Milestones +100 version 0.7.10 is fine. Anything above that causes the crash. I noticed another comment in Git that mentions you version 0.1.2 crashing with their version 0.7.11 and above.

Kind regards,

Oscar Capote

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by cepheidqq** [comment]
> The More Milestones +100 version 0.7.10 is fine. Anything above that causes the crash. I noticed another comment in Git that mentions you version 0.1.2 crashing with their version 0.7.11 and above.

try 0.1.3 with 0.7.11 :)

**Comment by Tropelage** [comment]
Works good now! :)

Kind regards,

Oscar Capote

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
